### ISSUE TITLE
Apps sometimes don’t remember logins between sessions

### ISSUE THREAD

**Issue by danielyli** [nativefier/nativefier#606]
### Description

For some sites, apps created using Nativefier don’t save authorization creds between sessions (app close-and-reopen cycles) despite explicitly checking the “Remember Me” box on those sites. Vanilla Chrome has no problems remembering logins for these sites.

### Steps to reproduce issue

This problem seems site-dependent. For instance, credentials are saved for GitHub but not Expensify or Greenhouse. Since reproduction requires valid credentials to specific sites, reproduction might be difficult unless you can find such a site.

On such a site, do the following:

  1.  Log in using your credentials, **remembering to check any Remember Me checkbox**.
  2.  Close the Nativefier app.
  3.  Reopen the app.

You should be presented with a login screen again.

### Details

- Are you nativefying a *public* website?
Public website with login

- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs?
`nativefier --help` doesn’t mention this.

- Full nativefier command used to build your app: `<command here>`
`nativefier --name Greenhouse --fast-quit --single-instance https://app.greenhouse.io/`

`nativefier --name Expensify --icon icns/expensify.icns --fast-quit --single-instance https://www.expensify.com/reports`

- Version of Nativefier (run `nativefier --version`): `v7.6.2`
- Version of node.js (run `node --version`): `v10.1.0`
- OS: `macOS High Sierra 10.13.4`
- Error message / stack trace (if any): None

**Comment by ronjouch** [comment]
@danielyli thanks for the feedback. Could your problem be a duplicate of https://github.com/jiahaog/nativefier/issues/164 ?

**Comment by danielyli** [comment]
It’s unknown if it is. In the cases I could find, I used an email+password combination for login, not credentials from Facebook, Google, etc. Unless these websites are using OAuth for email+password, they might not be related.

**Comment by awerlang** [comment]
Here is/was the same. No OAuth used. Credentials stored as cookies in the top-level domain.

I had to login every time I opened the app. Then using a tip from the OAuth thread, refreshing the page after the login, now it remembers the session next time. The trouble is that if I log out, it keeps me logged in next time.

Important to notice: this (forget session) happens when I specify a --name on the command line, even with no spaces.

**TL;DR** App generated with --name doesn't remember logins persisted in a parent domain. App generated with or without --name doesn't remembers log outs.

**Comment by bakeiro** [comment]
Same here

**Comment by fuzzy76** [comment]
@ronjouch do you still need more info on this?

**Comment by fuzzy76** [comment]
It seems fixed for me (I don't remember which specific sites I was having problems with).

**Comment by TheCleric** [comment]
Closing as we have one confirmation of a fix. If this still seems to be occurring we can reopen. 😄
